webapp/src/pages/FoodList.js

```diff
@@ -18,81 +18,80 @@ import { Link, useParams } from "react-router-dom";
 
 export default function FoodList() {
   const { offeringId } = useParams();
-  const getFoodOfferingList = React.useCallback(() => {
-    // TODO: Get all offerings information or make separate
-    // API call to get specific vendor offering list.
-    // Also, refactor the product loop in the return block below accordingly.
-    return api.getFoodOfferingList({ offeringId: offeringId || null });
+  const getFoodOfferingProducts = React.useCallback(() => {
+    return api.getFoodOfferingProducts({ offeringId: offeringId || null });
```

Comment:
What should we do for null? Seems like this would error?

Reply:
That shouldn't be there, good catch